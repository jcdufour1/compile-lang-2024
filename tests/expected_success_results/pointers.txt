8
9
10
9
19
39
part 67: 87/123
134/45
part 89: 134/1234
134/1298
134/1298
93/1298
134/1298
hello
bye
yes
pointer back to int: 0
pointer_to_pointer: yes
