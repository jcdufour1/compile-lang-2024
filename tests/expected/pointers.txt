8
9
19
87/123
134/45
